lib/node_modules/@stdlib/utils/group-in/docs/types/index.d.ts

```diff
@@ -108,7 +108,8 @@ type Indicator = Nullary | Unary | Binary;
 * var out = groupIn( obj, indicator );
 * // e.g., returns { 'b': [ 'beep', 'boop', 'bar' ], 'f': [ 'foo' ] }
 */
-declare function groupIn( obj: any, indicator: Indicator ): any;
+declare function groupIn<T, K>(obj: T, indicator: (v: T) => K): { [key: string]: K[] };
```

Comment:
These declarations do not look correct. The values returned by the indicator function are, TMK, not what is in the output arrays.

Furthermore, this PR should seek to enhance the existing `Indicator` definition, not simply ignore it as you have done here.